File: Character.cpp
class Character {
private:
    int HealthPoint;
    int movement;

public:
    Character(int healthPoint, int movement) : HealthPoint(healthPoint), movement(movement) {}

    int getHealthPoint() {
        return HealthPoint;
    }

    void setHealthPoint(int healthPoint) {
        HealthPoint = healthPoint;
    }

    int getMovement() {
        return movement;
    }

    void setMovement(int move) {
        movement = move;
    }
};

File: Character.java
public class Character {
    private int HealthPoint;
    private int movement;

    public Character(int HealthPoint, int movement) {
        this.HealthPoint = HealthPoint;
        this.movement = movement;
    }

    public int getHealthPoint() {
        return HealthPoint;
    }

    public void setHealthPoint(int HealthPoint) {
        this.HealthPoint = HealthPoint;
    }

    public int getmovement() {
        return movement;
    }

    public void setmovement(int movement) {
        this.movement = movement;
    }
}
